src/browser-bookmarks/import-bookmarks.js

```diff
@@ -0,0 +1,41 @@
+import db from '../pouchdb'
+import { updatePageSearchIndex } from '../search/find-pages'
+import { getVisitItemsForHistoryItems, transformToPageDoc, transformToVisitDoc,
+   convertHistoryToPagesAndVisits } from '../browser-history/import-history'
+import { isWorthRemembering, generatePageDocId, generateVisitDocId,
+  visitKeyPrefix, convertVisitDocId } from '../activity-logger'
+
+    function onRejected(error) {
+      console.log(`An error: ${error}`);
+    }
+
+    function getBookmarkItems() {
+      var bookmarkTree = browser.bookmarks.getRecent(100000);
```

Comment:
Also, though a 100k bookmarks sounds insane, let's go a bit further on the insane side. `importHistory` uses 9999999 as limit, could make it the same here.